infra/base-images/base-builder/compile_libfuzzer

```diff
@@ -16,16 +16,22 @@
 ################################################################################
 
 echo -n "Compiling libFuzzer to $LIB_FUZZING_ENGINE... "
-mkdir -p $WORK/libfuzzer
-pushd $WORK/libfuzzer > /dev/null
+export LIB_FUZZING_ENGINE="-fsanitize=fuzzer"
+if  [ "$SANITIZER" = "address" ] || [ "$SANITIZER" = "undefined" ]
+then
+  cp /usr/local/lib/clang/12.0.0/lib/linux/libclang_rt.fuzzer-$ARCHITECTURE.a \
```

Comment:
Can we make this more agnostic to the version rather than hardcoding 12.0.0?

Reply:
Done. The only caveat is the new version will simply fail if there is more than one clang installed.